Makefile

```diff
@@ -239,7 +239,7 @@ generate:
 
 generate_go: format_cpp
 	$(GO) generate ./...
-	$(GO) run github.com/vektra/mockery/v3@v3.5.0 --log-level="error"
+	$(GO) tool mockery --log-level="error"
```

Comment:
This is the main change. It uses the version from go.mod instead of the hardcoded.